data_model/master/clusters/OccupancySensing.xml

```diff
@@ -135,14 +135,6 @@ Davis, CA 95616, USA
       </field>
       <field id="1" name="HoldTimeMax" type="uint16">
         <mandatoryConform/>
-        <constraint>
```

Comment:
Is this because it can not handle the minOf and maxOf expressions yet?

Reply:
yup - need to update alchemy. We don't use these in tests currently.